spec/views/hq/orphans/show_spec.rb

```diff
@@ -1,17 +1,21 @@
 require 'rails_helper'
 
-RSpec.describe "hq/orphans/show.html.haml", type: :view do
+RSpec.describe "hq/orphans/show.html.erb", type: :view do
+  let(:orphan) { FactoryGirl.build_stubbed(:orphan) }
+
   describe 'the orphan exists' do
     before :each do
-      @orphan = build_stubbed :orphan
+      assign(:orphan, orphan)
       render
     end
 
     it 'should show the assigned orphan details' do
-      expect(rendered).to match @orphan.name
+      expect(rendered).to match orphan.name
```

Comment:
I can add more fields in the test, but at the end it's just to probe that something is shown.